ios/SettingsViewController.m

```diff
@@ -205,6 +205,14 @@ - (void)rerender {
                                 on:Settings.allowSkippingReviews
                             target:self
                             action:@selector(allowSkippingReviewsSwitchChanged:)]];
+  [model addItem:[[TKMSwitchModelItem alloc]
+                     initWithStyle:UITableViewCellStyleDefault
+                             title:@"Minimize review penalty"
+                          subtitle:@"Treat reviews answered incorrect multiple times as if "
```

Comment:
This subtitle does not seem to show in the UI.

Reply:
Fixed that - oops